src/ctx.cpp

```diff
@@ -88,11 +89,25 @@ pj_ctx pj_ctx::createDefault() {
 
     const char *projDebug = getenv("PROJ_DEBUG");
     if (projDebug != nullptr) {
-        const int debugLevel = atoi(projDebug);
-        if (debugLevel >= -PJ_LOG_TRACE)
-            ctx.debug_level = debugLevel;
-        else
-            ctx.debug_level = PJ_LOG_TRACE;
+        if (NS_PROJ::internal::ci_equal(projDebug, "ON")) {
+            ctx.debug_level = PJ_LOG_DEBUG;
+        } else if (NS_PROJ::internal::ci_equal(projDebug, "OFF")) {
+            ctx.debug_level = PJ_LOG_ERROR;
```

Comment:
In the documentation file it says
"and ``OFF`` as an alias for ``0``."
Should this be `PJ_LOG_NONE`? Or is doc file above wrong?
```suggestion
            ctx.debug_level = PJ_LOG_NONE;
```

Reply:
> Or is doc file above wrong?

ah yes, I intended OFF to be as the default, that is hide all debug and traces messages, but keep errors. Fixing the odc